### ISSUE TITLE
Release timeline for v1.8.11

### ISSUE THREAD

**Issue by cashpw** [orgzly/orgzly-android#1051]
This isn't so much a feature request as a request: Please release v1.8.11.

There's one issue remaining in the milestone (#1000). Can we move it to v1.8.12 and release v1.8.11? It's been ~1 year since the last merge and I think it'd be best to release the [work that's already been done](https://github.com/orgzly/orgzly-android/milestone/37?closed=1).

**Comment by cbkerr** [comment]
See the community fork Orgzly Revived with many merged fixes at https://github.com/orgzly-revived/orgzly-android-revived. Join us and spread the word!

There's in progress work on getting the forked package onto [F-Droid](https://gitlab.com/fdroid/rfp/-/issues/2562) and [Obtainium](https://github.com/orgzly-revived/orgzly-android-revived/issues/14).

**Comment by cashpw** [comment]
Glad to see the community has revived development on this great app! Unfortunately, I use a corp-managed device and can only install apps from the Play Store. I'll watch the [tracking issue for orgzly-revived](https://github.com/orgzly-revived/orgzly-android-revived/issues/35). I'd love to see the clock-in/clock-out feature land there!

I'm glad [orgzly-revived](https://github.com/orgzly-revived/orgzly-android-revived) is there for others through F-Droid et al.

**Comment by alensiljak** [comment]
The 1.8.11 (and 1.8.12) have been released!
The latter one is available on F-Droid.

**Comment by stefan2904** [comment]
> The 1.8.11 (and 1.8.12) have been released!

for the record, as of 2023-11-24 this comment is talking about orgzly-revived. orgzly is still at 1.8.10.


(just clarifying, since I just checked the Playstore and was confused  ;-)

**Comment by alensiljak** [comment]
> > The 1.8.11 (and 1.8.12) have been released!
> 
> for the record, as of 2023-11-24 this comment is talking about orgzly-revived. orgzly is still at 1.8.10.

True. And, as things currently stand (one year since the last commit or any signs of life in Orgzly) it looks like it's going to remain that way. Which is exactly the reason we went ahead with the Orgzly Revived project.
